src/irr/asset/CPLYMeshFileLoader.cpp

```diff
@@ -19,15 +19,81 @@ namespace irr
 namespace asset
 {
 
-// constructor
-CPLYMeshFileLoader::CPLYMeshFileLoader()
+_IRR_STATIC_INLINE_CONSTEXPR const char* VERT_SHADER_CACHE_KEY = "irr/builtin/shader/loaders/ply/vertex";
+_IRR_STATIC_INLINE_CONSTEXPR const char* FRAG_SHADER_CACHE_KEY = "irr/builtin/shader/loaders/ply/fragment";
```

Comment:
I want them to be shared between PLY, STL, and Serialized

we already have some simple solid shader, so you can copy what's done there.

Reply:
so needs to be created by asset manager since its shared and loader agnostic

Reply:
okay